presto-main/src/main/java/io/prestosql/sql/planner/RelationPlanner.java

```diff
@@ -277,6 +277,12 @@ protected RelationPlan visitSampledRelation(SampledRelation node, Void context)
         return new RelationPlan(planNode, analysis.getScope(node), subPlan.getFieldMappings());
     }
 
+    @Override
```

Comment:
would be also good to add catch-all visitNode to avoid being ashamed by NPE next time
like https://github.com/prestosql/presto/pull/3553/commits/034a787d90634462ea87c4ef68e201cb9e9f348c#diff-e7b6201f06648c5aa2d66f8e488cde68R145-R150 ;)